### PR TITLE
Add timestamp - timestamp -> interval day time Presto function (minus)

### PR DESCRIPTION
**Pull request by JasmineWeii**
Presto Timestamp subtract scalar function is missing, adding this function with the same implementation in Presto by comparing Timestamp milli seconds and return in interval day time.
https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/type/TimestampOperators.java#L64

### CODE REVIEW

File: velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp [hunk 1]
```diff
@@ -835,6 +835,26 @@ TEST_F(DateTimeFunctionsTest, plusMinusDateIntervalDayTime) {
...
+                         std::optional<Timestamp> t2) {
+    return evaluateOnce<int64_t>(
+        "c0 - c1",
+        makeRowVector({
+            makeNullableFlatVector<Timestamp>({t1}),
+            makeNullableFlatVector<Timestamp>({t2}),
+        }));
+  };
+  EXPECT_EQ(std::nullopt, minus(std::nullopt, std::nullopt));
+  EXPECT_EQ(1000, minus(Timestamp(1, 0), Timestamp(0, 0)));
```
Comment (mbasmanova) [inline velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp:849]: Since you only specifying seconds for timestamps, consider, simplifying this code to

```
EXPECT_EQ(1000, minus(1, 0));
EXPECT_EQ(-1000, minus(1, 2));
```

File: velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp [hunk 2]
```diff
@@ -835,6 +835,26 @@ TEST_F(DateTimeFunctionsTest, plusMinusDateIntervalDayTime) {
...
+  EXPECT_EQ(-1000, minus(Timestamp(1, 0), Timestamp(2, 0)));
+  EXPECT_THROW(
```
Comment (mbasmanova) [inline velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp:851]: Use VELOX_ASSERT_THROW

File: velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp [hunk 3]
```diff
@@ -835,6 +835,30 @@ TEST_F(DateTimeFunctionsTest, plusMinusDateIntervalDayTime) {
...
+                                             : std::optional<Timestamp>();
+    return evaluateOnce<int64_t>(
+        "c0 - c1",
+        makeRowVector({
+            makeNullableFlatVector<Timestamp>({timestamp1}),
+            makeNullableFlatVector<Timestamp>({timestamp2}),
+        }));
+  };
+
+  EXPECT_EQ(std::nullopt, minus(std::nullopt, std::nullopt));
```
Comment (mbasmanova) [inline velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp:852]: No need to test null values. These are handled by the framework.

**Review by mbasmanova — APPROVED** [review]
